### ISSUE TITLE
can't build WIPFC with error on helper/entit932.txt

### ISSUE THREAD

**Issue by graemeg** [open-watcom/open-watcom-v2#230]
I got the latest code, changed directory to the bld/wipfc/ directory, and followed the instructions in the INSTALL file. ./configure seem to have worked fine. 'make' run for a while and all seemed well, until near the end (so it seems). I then got this error.

```
In file included from cpp/xmp.cpp:37:
./cpp/brcmd.hpp:42:48: warning: explicitly assigning a variable of type 'Lexer *' to itself
      [-Wself-assign]
    Lexer::Token parse( Lexer* lexer ) { lexer = lexer; return document->getNextToken(); };
                                         ~~~~~ ^ ~~~~~
3 warnings generated.
mv -f .deps/xmp.Tpo .deps/xmp.Po
c++  -pipe -g -Wall    -o wipfc main.o compiler.o  document.o env.o ipfbuff.o  ipffile.o lexer.o bitmap.o  btmpblk.o cecmd.o cell.o  ctrlbtn.o ctrlgrp.o controls.o  errors.o extfiles.o fnt.o  fntcol.o fts.o gdword.o  gdict.o gnames.o header.o  index.o nls.o page.o strings.o  synonym.o toc.o tocref.o  util.o xref.o element.o text.o  entity.o punct.o whtspc.o  word.o tag.o acvwport.o  artlink.o artwork.o brcmd.o  caution.o cgraphic.o color.o  ctrl.o ctrldef.o ddf.o dl.o  docprof.o fig.o figcap.o fn.o  font.o hdref.o hide.o hn.o  hpn.o i1.o i2.o icmd.o  isyn.o lines.o link.o lm.o  lp.o note.o nt.o ol.o  p.o parml.o pbutton.o rm.o  sl.o table.o title.o ul.o  warning.o xmp.o  
document.o: In function `Document::makeBitmaps()':
/tmp/open-watcom-v2/bld/wipfc/cpp/document.cpp:536: warning: warning: tmpnam() possibly used unsafely; consider using mkstemp()
make[1]: don't know how to make helper/entit932.txt. Stop

make[1]: stopped in /tmp/open-watcom-v2/bld/wipfc
*** Error code 2

Stop.
make: stopped in /tmp/open-watcom-v2/bld/wipfc
```

It seems the makefile needs to be tweaked so as not to try and compile the helper/* files.

**Comment by jmalak** [comment]
Thanks for bug report. I will fix it.

**Comment by jmalak** [comment]
It is fixed by a833ba16539f8558760bb9b3002cba15a6315333 in git

**Comment by graemeg** [comment]
Many thanks, it works perfectly now. Finally I have a native FreeBSD 64-bit wipfc executable. :-)
